### ISSUE TITLE
Getting ArgumentError error after upgraded gem

### ISSUE THREAD

**Issue by jasnow** [reactjs/react-rails#695]
Help us help you! Please choose one:

- [X] My app crashes with `react-rails`, so I've included the stack trace and the exact steps which make it crash.
- [ ] My app doesn't crash, but I'm getting unexpected behavior. So, I've described the unexpected behavior and suggested a new behavior.
- [ ] I'm trying to use `react-rails` with another library, but I'm having trouble. I've described my JavaScript management setup (eg, Sprockets, Webpack...), how I'm trying to use this other library, and why it's not working.
- [ ] I have another issue to discuss.

-------

**Steps to reproduce**

 1. Clone my repo: https://github.com/jasnow/react-rails-tutorial

 2. Run "rake" to see that it runs cleanly.

 3. Then run "bundle update" to upgrade react-rails gem from 1.11.0 to 2.0.2.

 4. Then run "rake" and see the following error:
```
ArgumentError: A block is required to initialize a FileUpdateChecker
/Users/macbookpro/.rvm/gems/ruby-2.4.1@ruby2.4.1-rails5.1/gems/activesupport-5.1.0.rc1/lib/active_support/file_update_checker.rb:42:in `initialize'
/Users/macbookpro/.rvm/gems/ruby-2.4.1@ruby2.4.1-rails5.1/gems/react-rails-2.0.2/lib/react/rails/railtie.rb:36:in `new'
/Users/macbookpro/.rvm/gems/ruby-2.4.1@ruby2.4.1-rails5.1/gems/react-rails-2.0.2/lib/react/rails/railtie.rb:36:in `block in <class:Railtie>'
```
If you need more, please just add a comment and I will create a gist.

**Environment**

 1. Mac OS/X 10.12.4

 2. Ruby 2.4.1

 3. Rails 5.1.0.rc1

Feel free to add comments for more info.  Thanks.

**Comment by rmosolgo** [comment]
It looks like FileUpdateChecker started requiring a block in 5.1.0: https://github.com/rails/rails/pull/27824

**Comment by jasnow** [comment]
@rmosolgo - Thanks for the research.

**Comment by hundsim** [comment]
@rmosolgo Any guesses when this could be out of the way? 

I got the same error on a fresh Rails 5.1.0.rc1 installation, following the [three steps to install webpacker + react-rails](https://github.com/reactjs/react-rails#use-with-webpacker).

**Comment by rmosolgo** [comment]
This call needs an empty block: 

https://github.com/reactjs/react-rails/blob/master/lib/react/rails/railtie.rb#L36

Do want to add the block there? I don't mind following up by adding Rails 5.1.0 to our CI suite so that we keep compatibility in the future.

**Comment by hundsim** [comment]
Don't know if that was a friendly invitation to participate? In any case, I'm a noob and even if that one looked easy like a Sunday morning I better stay put. 

Btw, thx a bunch for your amazing work!

**Comment by rmosolgo** [comment]
Yes, it was an invitation to participate :) 

I'll take a look when I can, but if this issue is blocking you, I thought I'd share what I know so you could take a crack at fixing it. I think it literally needs two characters to run on Rails 5.1 (`{}`), so you could add them and push a new branch. 

The hard part will be getting 5.1 on CI, but if you pushed a branch with a fix, you could point your gemfile at that branch while master is getting worked out: 

```ruby 
gem "react-rails", github: "reactjs/react-rails", branch: "your-branch-here"
```

**Comment by jasnow** [comment]
Pointed my repo to your react-rails code and verified that the change fixed my issue. Thanks.

**Comment by rmosolgo** [comment]
Thanks for taking a look, just released it as 2.1.0!

**Comment by jasnow** [comment]
You are welcome.
Verified released version.
